From afdabc71c5fc5d9982d19ee7ef4bb8eed2cf9f74 Mon Sep 17 00:00:00 2001
From: Alexander Bokovoy <abokovoy@redhat.com>
Date: May 14 2021 08:38:47 +0000
Subject: Fix zone and forwarder registers to include dyndb-config.h


They kept includes for BIND's config.h which is not needed.

Signed-off-by: Alexander Bokovoy <abokovoy@redhat.com>

---

diff --git a/src/fwd_register.c b/src/fwd_register.c
index fcbd66f..5a3d4e2 100644
--- a/src/fwd_register.c
+++ b/src/fwd_register.c
@@ -6,7 +6,7 @@
 #include <isc/util.h>
 #include <dns/name.h>

-#include "config.h"
+#include "dyndb-config.h"
 #include "rbt_helper.h"
 #include "fwd_register.h"
 #include "util.h"
diff --git a/src/zone_register.c b/src/zone_register.c
index 28d9b7a..504aa9b 100644
--- a/src/zone_register.c
+++ b/src/zone_register.c
@@ -12,7 +12,7 @@
 #include <dns/result.h>
 #include <dns/zone.h>

-#include "config.h"
+#include "dyndb-config.h"
 #include "fs.h"
 #include "ldap_driver.h"
 #include "log.h"
